upgrading/automated_upgrades.adoc

```diff
@@ -193,6 +197,11 @@ an infrastructure node to your cluster.
 xref:../admin_guide/manage_nodes.adoc#updating-labels-on-nodes[marking it]
 with the `openshift_cluster_monitoring_operator_node_selector`.
 
+.. If you use the standard {product-title} registry, prepare for the change from
```

Comment:
@jiajliu, does this sound ok for the registry change?

Reply:
@kalexand-rh 
This change seems not updated in the pr. Could u show me the new open pr for it.

Reply:
It's in the PR but not in the preview because I did not manually update it first. I made a new PR here for follow-up requests: https://github.com/openshift/openshift-docs/pull/12355

The change should be there.